dev-libs/libb64/patches/libb64-20170101.patchset

```diff
@@ -0,0 +1,22 @@
+From 5f76b7d8246a06b44909add406a1b64461def572 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Zolt=C3=A1n=20Mizsei?= <zmizsei@extrowerk.com>
+Date: Sun, 24 May 2020 21:46:39 +0200
+Subject: Build shared lib
+
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index d51293f..3f80228 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -3,7 +3,7 @@ project(b64 C)
+ 
+ include_directories(include)
+ 
+-add_library(${PROJECT_NAME} STATIC
++add_library(${PROJECT_NAME} SHARED
```

Comment:
If they didn't intend this be shipped as a shared library, maybe it shouldn't be. What do other distros do?

Reply:
I was unable to build Transmission with the in-tree libs, and 3.00 tarball doesn't contains these libs (i was later informed that one supposed to use the .xz release for that).
I had the idea maybe other ports can also profit if we build them as shared, because B64 and DHT used in other software too.
The libs are updated separately from transmission, with shared libs we can integrate the changes from upstream more easily.
Somebody have to decide.

Reply:
Well, again, what do other distros do? If they ship these as shared, then OK; but I'd prefer we not diverge too far here.

Reply:
https://repology.org/project/libutp/versions
https://repology.org/project/libb64/versions

Reply:
On a quick look atleast ARCH is building a shared lib for this one too? https://git.archlinux.org/svntogit/community.git/tree/trunk/PKGBUILD?h=packages/libb64